src/publishProject/projectUtils.ts

```diff
@@ -107,66 +112,31 @@ export async function readProjectProperties(
             return undefined;
         }
         const version = await getProjectTargetVersion(sqlProjectsService, projectFilePath);
-        const props: ProjectProperties = {
-            projectGuid: result.projectGuid,
-            configuration: result.configuration,
-            outputPath: result.outputPath,
-            databaseSource: result.databaseSource,
-            defaultCollation: result.defaultCollation,
-            databaseSchemaProvider: result.databaseSchemaProvider,
-            projectStyle: result.projectStyle,
+        return {
+            ...result,
             targetVersion: version,
         };
-        return props;
     } catch {
         return undefined;
     }
 }
 
+/**
+ * Gets the appropriate server name display string based on the target version.
+ * @param target - The target version string (e.g. "AzureV12", "150")
+ * @returns "Azure SQL server" for Azure SQL Database, "SQL server" for all other targets
+ */
 export function getPublishServerName(target: string) {
-    return target ===
-        targetPlatformToVersion.get("Azure SQL Database" /* SqlTargetPlatform.sqlAzure */)
+    return target === targetPlatformToVersion.get(SqlTargetPlatform.sqlAzure)
```

Comment:
Out of curiosity, what is the target platform value for Fabric?  Is that still `sqlAzure`?